services/federator/test/unit/Test/Federator/ExternalServer.hs

```diff
@@ -32,6 +37,10 @@ import Wire.API.Federation.GRPC.Types
 
 genMock ''Brig
 
+genMock ''Log.TinyLog
```

Comment:
You shouldn't need to mock the logging monad, but simply ignore it (unless you actually want to test that things get logged). So something like:

```haskell
noLogs :: Sem (Log.TinyLog ': r) a -> Sem r a
noLogs = interpret f
  where
    f :: Applicative n => Log.TinyLog m x -> n x
    f (Log.Polylog _ _) = pure ()
```

and then use `noLogs` instead of `runTinyLog l`.

Reply:
Thanks!